modules/nw-ipfailover-remove.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ipfailover.adoc
+
+[id="nw-ipfailover-remove_{context}"]
+= Removing IP failover
+
+When IP failover is initially configured, the worker nodes in the cluster are modified with an `iptables` rule that explicitly allows multicast packets on `224.0.0.18` for Keepalived. Because of the change to the nodes, removing IP failover requires running a job to remove the `iptables` rule.
+
```

Comment:
the cleanup does more than that, remove kernel module, remove all configured keepalived VIPs you decided to use general term here or go into details but it has to be accruate , I suggest general term like remove all node's configuration  resulted  from  deploying ipfailover or something like that

Reply:
I'll leave the kernel module out of it because that seems pretty invisible and also unlikely to be any kind of a clue to tech support.  I'll include mention of the virtual IPs because that could be a troubleshooting clue for tech support to notice--in the event of any trouble, though none expected.